gravitee-apim-definition/gravitee-apim-definition-jackson/src/main/java/io/gravitee/definition/jackson/datatype/api/deser/ApiDeserializer.java

```diff
@@ -254,6 +254,12 @@ public T deserialize(JsonParser jp, DeserializationContext ctxt, T api, JsonNode
             api.setDefinitionContext(definitionContext);
         }
 
+        JsonNode stateNode = node.get("state");
```

Comment:
IMHO, we shouldn't add the state on the api definition itself. It should not be part of it and instead managed in another way.